### DISCUSSION TITLE
Documentation code sections color scheme

### DISCUSSION THREAD

**Discussion by dvesh3** [pimcore/pimcore#13236] [Polls]
Hello Pimconauts 👨‍🚀 👩‍🚀 

Currently, we are in the process of improving the Pimcore documentation theme and since, you guys are looking into the docs most the time, your opinion matters the most.

In this poll, we would like to ask you to choose a color scheme for code sections:

## 1. Light: light background with dark text
![image](https://user-images.githubusercontent.com/5137917/191932318-df078aa5-c4d1-4c36-899b-a54f09a42326.png)

## 2. Dark: Dark background with light text
![image](https://user-images.githubusercontent.com/5137917/191931956-88ce1a98-ce28-4ccd-a3fd-f6a3acaef3a3.png)

Thanks for your vote 💜

**Comment by NiklasBr** [comment]
[This is a good argument why light-on-dark should be avoided](https://tidbits.com/2019/05/31/the-dark-side-of-dark-mode/). as well as [this article](https://medium.com/geekculture/dark-mode-has-worsened-my-sight-de54b8fe0455).

I made the original PR which was accepted and then rejected because I found the light-on-dark to be both of too low contrast and hard to read. It was made worse that the code blocks were dark on an otherwise white page.

**Comment by SamyMP** [comment]
Why not do both and let the user choose ? As it is done here for example (top right of the screen) : **https://sequelize.org/docs/v6/**

**Reply by jdreesen** [reply]
I'd love to have a dark mode (not only the code blocks), too.

**Reply by NiklasBr** [reply]
Yes, a [prefers-color-scheme CSS ](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme)would resolve it pretty much universally. I could do a PR with that if it is acceptable.

**Reply by SamyMP** [reply]
That would be great

**Reply by dvesh3** [reply]
yes, that's a good suggestion 👍  we are happy to receive a PR for the switch 😊
